x-pack/plugins/monitoring/server/lib/logstash/__tests__/get_pipeline.js

```diff
@@ -15,7 +15,7 @@ describe('get_pipeline', () => {
     let vertex;
     let vertexStatsBucket;
     let totalProcessorsDurationInMillis;
-    let timeboundsInMillis;
+    let timeboundsInSeconds;
```

Comment:
Good catch! While you're renaming it, do you mind renaming it to `timeseriesIntervalInSeconds` so it matches the function definition?